src/components/App/App.js

```diff
@@ -0,0 +1,21 @@
+import React, { useState, useEffect } from 'react';
+import { getTips } from '../../ApiCalls'
+import TipJar from '../TipJar/TipJar'
+
+
+const App = () => {
+  const [advice, setAdvice] = useState([]);
+
+  useEffect(() => {
```

Comment:
Great use of this useEffect with the fetch request and to update state. Love that you're diving into react hooks. Perhaps we could try some async and await here.